definitions/npm/normalizr_v3.x.x/flow_v0.25.x-/normalizr_v3.x.x.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
```

Comment:
Please link to source `.ts` file for convenience and future referencing.

Reply:
Done. Added jsdoc comment with permanent link to normalizr's github repo